packages/website/src/components/editor/useSandboxServices.ts

```diff
@@ -76,8 +76,14 @@ export const useSandboxServices = (
           colorMode === 'dark' ? 'vs-dark' : 'vs-light',
         );
 
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-member-access
+        const libs = ((window.ts as any).libs as string[]) ?? ['esnext'];
```

Comment:
`libs` is not exposed publicly, but we need a way to retrieve list of libs that are valid for specified typescript version

Reply:
this change makes this code backward and forward compatible as long as ts is going to have this field exposed

Reply:
alternatively we could define list of libs that should be allowed per ts version, but its easy to make mistake in there